### ISSUE TITLE
Is there validation(test) dataset for Asian? 

### ISSUE THREAD

**Issue by dypromise** [deepinsight/insightface#609]
Dear all!
I want to fine-tune a face recognition model for Asian, and I plan to use author's pre-trained model(ResNet50-arcface-msra1m) as base model and to fine-tune it  on glint-asian dataset. My confusion is that how to evaluate this asian model? use LFW/CFP/AgeDB? I think if my target is to recognize Asian people, just using LFW/CFP/AgeDB as validation dataset is not fair. Am I correct? or it is not necessary to use a validation set which just contains Asian people? or I 
can split glint-asian dataset to new train_set and val_set? which one is best? Any other idea? Appreciate any reply!

**Comment by SueeH** [comment]
Same situation！
Now i  fine-tune on deep-glint, and the train accuracy stay about 47%，but lfw/cfp_fp/ageddb_30 acc keep about 99%/93%/97%...I have tried decay wd and lr，but not work.
So could you share your finetuning process， and any advice ？
Thanks in advance

**Comment by KhurramPirov** [comment]
> Same situation！
> Now i fine-tune on deep-glint, and the train accuracy stay about 47%，but lfw/cfp_fp/ageddb_30 acc keep about 99%/93%/97%...I have tried decay wd and lr，but not work.
> So could you share your finetuning process， and any advice ？
> Thanks in advance

Did you try to preprocess and detect all images  from deep-glint by retina face ?

**Comment by SueeH** [comment]
> Did you try to preprocess and detect all images from deep-glint by retina face ?

1.I downloaded and used the data provided by author(.rec) directly；
2.the results shown before was trained using combined loss, when using arc loss, training acc =  99%, and lfw/cfp_fp/ageddb_30 acc reached 99.8%,96.5%,98.2%.
3.But how evaluate the new model (finetuning on glint), the above three dataset or any other dataset？

**Comment by KhurramPirov** [comment]
Change path in config.py. For example in emore dataset string to your dataset.

**Comment by Vietdung113** [comment]
> > Did you try to preprocess and detect all images from deep-glint by retina face ?
> 
> 1.I downloaded and used the data provided by author(.rec) directly；
> 2.the results shown before was trained using combined loss, when using arc loss, training acc = 99%, and lfw/cfp_fp/ageddb_30 acc reached 99.8%,96.5%,98.2%.
> 3.But how evaluate the new model (finetuning on glint), the above three dataset or any other dataset？

How can u reach 99% accuracy on training set with arc loss?
Can u shared your training plan and config?

**Comment by SueeH** [comment]
> > > Did you try to preprocess and detect all images from deep-glint by retina face ?
> > 
> > 
> > 1.I downloaded and used the data provided by author(.rec) directly；
> > 2.the results shown before was trained using combined loss, when using arc loss, training acc = 99%, and lfw/cfp_fp/ageddb_30 acc reached 99.8%,96.5%,98.2%.
> > 3.But how evaluate the new model (finetuning on glint), the above three dataset or any other dataset？
> 
> How can u reach 99% accuracy on training set with arc loss?
> Can u shared your training plan and config?

1.I think the high training accuracy is due to last softmax-layer overfitting, since the test accuracy on lfw or my data almost keep same with original model.
2.same config with author, lr=0.01,lr_step='10e4,15e4,20e4',with each step lr=lr*0.5,after 8 epoch(2.5e4 iters) train_acc:~0.83 then use lower acc retrain....
